src/app/clusters/mode-base-server/mode-base-server.cpp

```diff
@@ -38,6 +38,22 @@ namespace app {
 namespace Clusters {
 namespace ModeBase {
 
+void Instance::RegisterInstance(Instance * instance)
+{
+    if (!gModeBaseAliasesInstances.Contains(instance))
+    {
+        gModeBaseAliasesInstances.PushBack(instance);
+    }
+}
+
+void Instance::UnregisterInstance(Instance * instance)
+{
+    if (gModeBaseAliasesInstances.Contains(instance))
+    {
+        gModeBaseAliasesInstances.Remove(instance);
```

Comment:
I think Remove() is safe to call even if the instance is not in the list, and that will save the work of checking whether it is....

Reply:
OK, I removed the check.